custom/export_demographics.php

```diff
@@ -1,23 +0,0 @@
-<html>
```

Comment:
@sunsetsystems 
Hi Rod, looks like this and the custom/export_labworks.php were added by you in 2005. Is this still being used or can we remove it?
thanks, -brady